eclipse-collections/src/main/java/org/eclipse/collections/impl/utility/internal/InternalArrayIterate.java

```diff
@@ -1214,4 +1226,108 @@ public static <R extends Collection<T>, T> R rejectWithIndex(
         }
         return target;
     }
+
+    public static <R extends MutableBooleanCollection, T> R collectBoolean(
```

Comment:
I wonder if there's any point in passing the size explicitly - looks like we can deduct it from items.length directly? In the appropriate places we have a null check so we can reduce the param list?

Reply:
got u - my bad, thx both for the explanation